parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -209,6 +212,19 @@ public static boolean getEnableDictionary(JobContext jobContext) {
     return getEnableDictionary(getConfiguration(jobContext));
   }
 
+  public static String getBloomFilterColumnNames(Configuration conf) {
```

Comment:
Why separate getBloomFilterColumnNames from getBloomFilterExpectedNDV, and why use Strings? Why not just directly return a map and have a single method, getBloomFilterColumnExpectedNDVs?